### PR TITLE
[WIP]Hands within hands

### PR DESCRIPTION
**Pull request by IK3I**
Ports [tg 20283](https://github.com/tgstation/tgstation/pull/20283)

Fixes up hand code while also allowing admeme shenanigans with hand counts. This is almost entirely backend stuff.

:cl:
tweak: Hands are now lists
rscadd: You can now have as many hands as an admeme deems you capable of using.
/:cl:

### CODE REVIEW

**Comment by tigercat2000** [comment]
aaaaaaaaaa

**Comment by IK3I** [comment]
and no, this doesn't yet compile.

**Comment by Aurorablade** [comment]
you know...TG has something like this...

**Comment by taukausanake** [comment]
> you know...TG has something like this...

> Ports tg 20283

:confused:

So does this give you more hand inventory icons or hand inventories that you can jump/switch between?
Does it do anything to mob sprites?
If I have four hands then can I choke two people at the same time? (because you need two hands for grab)
Can Wryn use their baby hands that they are totally supposed to have to carry more things?

240 files is way too much to read through to answer these questions

**Comment by IK3I** [comment]
first off, there's no frontend change. Admins would have to varedit and rebuild your hands for anything to happen. Second, The gui fully supports as many hand slots as you may want.

**Comment by Quantum-M** [comment]
https://www.homestuck.com/problem-sleuth/1549

**Comment by Citinited** [comment]
I've got to hand it to you, this looks pretty fantastic. It'd be pretty handy to have this in-game.

**Comment by Birdtalon** [comment]
This is the best PR in a long time hands down.

**Comment by Superhats** [comment]
Possible Kidan buff?

**Comment by alexkar598** [comment]
@IK3I can you set a mob to have 0 hands?

**Comment by IK3I** [comment]
@alexkar598 The default mob has zero hand slots

File: paradise.dme [hunk 1]
```diff
@@ -361,7 +361,6 @@
 #include "code\game\alternate_appearance.dm"
 #include "code\game\asteroid.dm"
 #include "code\game\atoms.dm"
-#include "code\game\atoms_movable.dm"
```
[the remark this answers is not in the document]
Comment (IK3I) [reply paradise.dme]: That's why its screaming about anchors all the time! I love debugging

**Comment by variableundefined** [comment]
Unfortunately we'll be closing this PR as it has been stale for months. You are free to re-open this when it is ready. If this is re-opened during a feature freeze, no token will be charged.

**Pull request closed without merging**